src/main/webapp/res/js/common.js

```diff
@@ -0,0 +1,17 @@
+const juniorElem = document.querySelector('.junior-class');
+const intermediateElem = document.querySelector('.intermediate-class');
+const advancedElem = document.querySelector('.advanced-class');
+const masterElem = document.querySelector('.master-class');
+
+juniorElem.addEventListener('click', () => {
+    location.href="/level/junior";
+});
+intermediateElem.addEventListener('click', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/koreait0000/NONG.4/pullRequest?prid=7788346)
```suggestion
intermediateElem.addEventListener("click", () => {
```